saml2/saml2-service-provider/src/test/java/org/springframework/security/saml2/provider/service/registration/OpenSamlRelyingPartyRegistrationBuilderHttpMessageConverterTests.java

```diff
@@ -114,31 +112,6 @@ public void readWhenDescriptorFullySpecifiedThenConfigures() throws Exception {
 				.isEqualTo(x509Certificate(CERTIFICATE));
 	}
 
-	// gh-9051
```

Comment:
Can you explain why these tests were removed?

Reply:
Sorry for that. I will be reverting the class to how it was